src/map/ai/states/item_state.cpp

```diff
@@ -240,7 +240,7 @@ void CItemState::TryInterrupt(CBattleEntity* PTarget)
         UpdateTarget(m_PEntity->IsValidTarget(m_targid, m_PItem->getValidTarget(), m_errorMsg));
     }
 
-    uint16 msg = 445; // you cannot use items at this time
+    uint16 msg = 62; // the item fails to activate
 
     if (HasMoved() || m_PEntity->StatusEffectContainer->HasPreventActionEffect())
     {
```

Comment:
instead of changing the default message, change the message used inside this block.
```c++
if (HasMoved() || m_PEntity->StatusEffectContainer->HasPreventActionEffect())
{
    msg                = MSGBASIC_ITEM_FAILS_TO_ACTIVATE;
    m_interrupted = true;
}